File: src/server/Adfnet.Service/Models/ParameterModel.cs
using System;
using Adfnet.Core;
using Adfnet.Core.ValueObjects;

namespace Adfnet.Service.Models
{
    public class ParameterModel : IServiceModel
    {
        public Guid Id { get; set; }
        public int DisplayOrder { get; set; }
        public bool IsApproved { get; set; }
        public int Version { get; set; }
        public DateTime CreationTime { get; set; }
        public IdName Creator { get; set; }
        public DateTime LastModificationTime { get; set; }
        public IdName LastModifier { get; set; }
        public string Key { get; set; }
        public string Value { get; set; }
        public bool Erasable { get; set; }
        public string Description { get; set; }
        public IdCodeName ParameterGroup { get; set; }

    }
}

File: src/client/web/private/src/app/models/parameter-model.ts
import { IdCodeName } from '../value-objects/id-code-name';
import { IdName } from '../value-objects/id-name';

export class ParameterModel {
    id: string;
    displayOrder: number;
    isApproved: boolean;
    version: number;
    creationTime: Date;
    creator: IdName;
    lastModificationTime: Date;
    lastModifier: IdName;
    key: string;
    value: string;
    erasable: boolean;
    description: string;
    parameterGroup: IdCodeName;
    constructor() {
        this.creator = new IdName();
        this.lastModifier = new IdName();
        this.parameterGroup = new IdCodeName();
    }
}
